### DISCUSSION TITLE
使用vw方案时，行内样式px不能转vw怎么解决

### DISCUSSION THREAD

**Discussion by taojunnan** [youzan/vant#10322] [Q&A]
使用postcss-px-to-viewport将px转vw，vant中用class写的样式能把px转成vw，但是行内样式的px就不能转，请问这种情况怎么解决

**Comment by yuningjiang123** [comment]
无法解决   postcss-px-to-viewport这玩意儿只能处理css中的样式   无法处理行内样式

**Comment by jianfei0220** [comment]
请问有解决吗？vant部分组件的样式，并没有转换为vw
